src/webdriver/index.ts

```diff
@@ -98,11 +98,14 @@ export function augmentDriver<T>(driver: WebDriver, baseUrl: string) {
         })
         .perform();
     },
+    
+    // type ValidType = string | boolean | Record<string, ValidType> | ValidType[]
```

Comment:
? how does this work with this commented out?

Reply:
I just dropped this a s a sketch real quick, I'd like to get to it, I think should be easy

Reply:
might want to put in a comment about what it's for, otherwise it looks like dead code

Reply:
I am saying, I need to work on this pr. i just put some very early draft on :P. I literally modified some files to explain what I had in mind and pushed